app/views/media/_editor_render.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_tag :figure, class: "image #{image[:align]}" do  %>
+  <%= content_tag :span, clas: 'img' do %>
```

Comment:
__[Misspell]__

"clas" is a misspelling of "class"


<!-- Comment by [SideCI](https://sideci.com) -->